sites/partners/__tests__/components/listings/PaperListingForm/sections/RankingsAndResults.test.tsx

```diff
@@ -0,0 +1,186 @@
+    await userEvent.click(waitlistYesRadio)
+
+    await screen.findByText("How is the application review order determined?")
+
+    const lotteryRadio = document.getElementById("reviewOrderLottery") as HTMLInputElement
+    await userEvent.click(lotteryRadio)
+
+    await waitFor(
+      () => {
+        expect(screen.queryByTestId("lottery-start-date")).toBeInTheDocument()
+      },
+      { timeout: 5000 }
+    )
+
+    expect(screen.getByTestId("lottery-start-time")).toBeInTheDocument()
+    expect(screen.getByTestId("lottery-end-time")).toBeInTheDocument()
+  })
+
+  it("should not show lottery fields when enableWaitlistLottery is false and waitlist is open", async () => {
+    document.cookie = "access-token-available=True"
+    server.use(
+      rest.get("http://localhost/api/adapter/user", (_req, res, ctx) => {
+        return res(ctx.json(adminUserWithoutWaitlistLotteryFlag))
+      })
+    )
+
+    render(
+      <FormComponent
+        values={{
+          ...formDefaults,
+          jurisdictions: { id: "jurisdiction1" },
+          listingAvailabilityQuestion: "openWaitlist",
+        }}
+      >
+        <RankingsAndResults
+          requiredFields={[]}
+          whatToExpectEditor={null as any}
+          whatToExpectAdditionalTextEditor={null as any}
+        />
+      </FormComponent>
+    )
+
+    await screen.findByText("Rankings & results")
+
+    const waitlistYesRadio = await screen.findByRole("radio", { name: "Yes" })
+    await userEvent.click(waitlistYesRadio)
+
+    expect(screen.queryByText("How is the application review order determined?")).not.toBeInTheDocument()
+
+    expect(screen.queryByTestId("lottery-start-date")).not.toBeInTheDocument()
+    expect(screen.queryByTestId("lottery-start-time")).not.toBeInTheDocument()
+    expect(screen.queryByTestId("lottery-end-time")).not.toBeInTheDocument()
+  })
+
+  it("should not show lottery fields when waitlist is not open even with feature flag enabled", async () => {
+    document.cookie = "access-token-available=True"
+    server.use(
+      rest.get("http://localhost/api/adapter/user", (_req, res, ctx) => {
+        return res(ctx.json(adminUserWithWaitlistLotteryFlag))
+      })
+    )
+
+    render(
+      <FormComponent
+        values={{
+          ...formDefaults,
+          jurisdictions: { id: "jurisdiction1" },
+          listingAvailabilityQuestion: "openWaitlist",
+        }}
+      >
+        <RankingsAndResults
+          requiredFields={[]}
+          whatToExpectEditor={null as any}
+          whatToExpectAdditionalTextEditor={null as any}
+        />
+      </FormComponent>
+    )
+
+    await screen.findByText("Rankings & results")
+
+    const waitlistYesRadio = document.getElementById("waitlistOpenYes") as HTMLInputElement
+    await userEvent.click(waitlistYesRadio)
+
+    await waitFor(() => {
+      expect(screen.getByText("How is the application review order determined?")).toBeInTheDocument()
```

Comment:
suggestion: you should be able to remove the `waitFor` by switching to `findByText`

```
    expect(
      await screen.findByText("How is the application review order determined?")
    ).toBeInTheDocument()
```